mlflow/paddle/__init__.py

```diff
@@ -196,7 +195,10 @@ def forward(self, inputs):
     model_data_subpath = "model"
     output_path = os.path.join(path, model_data_subpath)
 
-    paddle.jit.save(pd_model, output_path)
+    if isinstance(pd_model, paddle.Model):
+        pd_model.save(output_path, training=False)
```

Comment:
Does `training=False` here mean saving a model in the evaluation mode?

Reply:
Yes, it means that the model saved will be used for inference only and no longer support a second training process.

Reply:
I see, is it possible to load a model saved with `training=False` and re-train it?

Reply:
No. If we want to re-train it, we should set ```training=True```

Reply:
Got it, would it make sense to make `training` configurable via an argument to allow re-training?

```diff
def save_model(
    ...
+   training=False
)
```

Reply:
It makes sense. I will add a new example and unit tests  with the flag ```training=True```.

Reply:
HI @harupy 
Now it supports model retrain. I also add a new example that is trained using high level paddle api. Please take a look at it. Thank you so much